zulipterminal/themes/Common.py

```diff
@@ -0,0 +1,93 @@
+"""
+COMMONs File
+------------
+Contains color definitions or styles common across all themes.
+For further details on themefiles look at the theme contribution guide.
+"""
+from enum import Enum
+
+from zulipterminal.helper import color_properties
+
+
+# fmt: off
+class DefaultColor(Enum):
+    # color       =  16code          256code    24code
+    DEFAULT       = 'default         default    default'
+    BLACK         = 'black           g19        default'
+    DARK_RED      = 'dark_red        #a00       default'
+    DARK_GREEN    = 'dark_green      #080       default'
+    BROWN         = 'brown           #880       default'
+    DARK_BLUE     = 'dark_blue       #24a       default'
+    DARK_MAGENTA  = 'dark_magenta    h90        #870087'
+    DARK_CYAN     = 'dark_cyan       #088       default'
+    DARK_GRAY     = 'dark_gray       #666       default'
+    LIGHT_RED     = 'light_red       #f00       default'
+    LIGHT_GREEN   = 'light_green     #0f0       default'
+    YELLOW        = 'yellow          #ff0       default'
+    LIGHT_BLUE    = 'light_blue      #28d       default'
+    LIGHT_MAGENTA = 'light_magenta   #c8f       default'
+    LIGHT_CYAN    = 'light_cyan      h152       #afd7d7'
```

Comment:
Since zt_blue used the 16 bit `light_cyan` and `dark_magenta`, I went searching for 256 color codes [here](https://jonasjacek.github.io/colors/). It also had 24bit codes so I used it.

Re "default", It just uses the default foreground or background, so it looks mono with stream colors. We can add proper colors later. It's not related to this PR I think.

The tests don't check if the color codes are correct just that there are 3 space separated words. We should use a **regex** match to check the format maybe.

Reply:
I've added regex tests as well